Dockerfile

```diff
@@ -20,6 +20,7 @@ RUN URL=https://hndl.urbackup.org/Server/${VERSION} && \
     && echo "deb $dry\ndeb-src $dry" >/etc/apt/sources.list.d/${DEBIAN}-backports.list \
     && export DEBIAN_FRONTEND=noninteractive \
     && apt-get update \
+    && apt-get libguestfs-tools \
```

Comment:
`install` missing. Maybe the package just had to be added inside the line below.

Reply:
You right install should be added, maybe
@uroni know better how to add it..

Reply:
@Commifreak I'm made xml template for Unraid for this container, but founded some stuff like image mounting not enabled by default...